src/main/java/it/gov/pagopa/pu/pagopapayments/registry/RegistryLogger.java

```diff
@@ -67,6 +87,9 @@ public <I, O> O execute(
     try {
       response2outcome = requestHandler.get();
     } catch (Exception e) {
+      if (throwException) {
+        throw e;
+      }
       response2outcome = Triple.of(exceptionHandler.apply(e), null, RegistryOutcome.KO);
```

Comment:
nope! at the moment, let's just reason on exceptionHandler != null instead of introduce a new parameter...
I will think on how to handle it on the other tasks...

The registry should log the response... with this handling, you are not logging anything (the finally block will goes into error...
```suggestion
      if(expcetionHandler != null) {
         response2outcome = Triple.of(exceptionHandler.apply(e), null, RegistryOutcome.KO);
      } else {
         response2outcome = Triple.of(null, null, RegistryOutcome.KO);
         throw e;
      }
```

Reply:
This has not been resolved